Tomlet/TomlSerializationMethods.cs

```diff
@@ -136,7 +136,7 @@ internal static Deserialize<object> GetCompositeDeserializer(Type type)
             else
             {
                 //Get all instance fields
-                var fields = type.GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance);
+                var fields = type.GetFields(BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.SetProperty);
```

Comment:
I don't think this change is required, `SetProperty` only applies when calling methods, from what I can see on the docs, and regardless your change isn't required by the code you wrote below.